### ISSUE TITLE
[0.9.4.2] Incorrect logrotate config permissions with Debian package installation

### ISSUE THREAD

**Issue by immunda** [influxdb/influxdb#4523]
Influx ver: `0.9.4.2`
OS ver: `Ubuntu 14.04.2 LTS`
Package: `http://influxdb.s3.amazonaws.com/influxdb_0.9.4.2_amd64.deb`

Hi folks,

I noticed I was getting mammoth logfile usage and it wasn't being rotated as expected. I ran logrotate as debug and it failed, giving me a permissions error:

```
user@xxx:~$ sudo logrotate -d /etc/logrotate.d/influxd 
Ignoring /etc/logrotate.d/influxd because of bad file mode.

Handling 0 logs
```

The permissions:

```
user@xxx:~$ ls -la /etc/logrotate.d/influxd 
-rw-rw-r-- 1 root root 113 Sep 29 17:51 /etc/logrotate.d/influxd
```

The following permission change corrected the problem, and a force run of `logrotate` on the config ran successfully.

```
user@xxx:~$ sudo chmod g-w /etc/logrotate.d/influxd
```

Thanks for all the ongoing work! Phil.

**Comment by rossmcdonald** [comment]
Duplicate of #4446, resolved in #4447

**Comment by immunda** [comment]
🙏Appreciated, look forward to the next release.
